exercises/target/example.js

```diff
@@ -0,0 +1,17 @@
+export default function solve(x, y) {
+  // Define as numbers
+  x = Number(x);
```

Comment:
I think we should avoid re-assigning the input params.  I'm surprised the linter didn't complain about this. We're currently using the Airbnb style guide:  http://airbnb.io/javascript/#functions--mutate-params